requirements.txt

```diff
@@ -16,3 +16,5 @@ transformers == 4.37.2
 torch == 2.2.0
 pandas == 2.0.0
 nltk == 3.8.1
+spacy == 3.7.4
+en_core_web_sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.7.1/en_core_web_sm-3.7.1-py3-none-any.whl
```

Comment:
Is this package a model? what size is it?

Reply:
Yes, the size of model is 12.2MB. More details: https://github.com/explosion/spacy-models/releases